### DISCUSSION TITLE
Generate a raster tessellation

### DISCUSSION THREAD

**Discussion by damulaoshi** [neperfepx/neper#173] [Q&A] [answered]
Dear Romain,
Based on my understanding, if I want to generate voxel-based grains rather than using Voronoi tessellation, I should use the command 'tesr', raster tessellation in Neper? Am I right or no?

If I am right, how can I generate a model by using 'tesr' command in Module-T?
If I am not right, is there any possibility to generate voxel-based microstructures? And what's the command like? 

For example, I wrote the command:
```
neper -T -n from_morpho -domain "cube(10,10,10)" -morpho "diameq:lognormal(0.5,0.02)" -ori random -for tess,ori -o test
```

How can I add 'tesr' to my command?

Thank you before and wish to hear your answer.
Best regards,
Damu

**Comment by rquey** [comment, marked as answer]
It's as simple as [`-format tesr`](https://neper.info/generation.html#scalar-and-raster-formats).

**Reply by damulaoshi** [reply]
Ok, thank you so much!
